features/step_definitions/file_steps.coffee

```diff
@@ -26,6 +27,13 @@ module.exports = ->
     addToJsonFile filePath, content, done
 
 
+  @Given /^I have configured "([^"]*)" to be (.+)$/, (key, value, done) ->
+    filePath = path.join @tmpDir, 'dependency-lint.json'
+    content = {}
+    content[key] = coffee.eval(value)
+    addToJsonFile filePath, content, done
```

Comment:
FYI, coffeescript 1.9.1 allows this to be
```coffeescript
content = "#{key}": coffee.eval(value)
addToJsonFile filePath, content, done
```

In the interest of consistency I kept it the same as the rest of the step definitions, but maybe a future PR can change these